src/main/java/com/hillel/items_exchange/dto/CategoriesNamesDto.java

```diff
@@ -0,0 +1,16 @@
+package com.hillel.items_exchange.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class CategoriesNamesDto {
```

Comment:
I don't see a reason to use this class instead of returning Categories' names directly.